Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -302,6 +328,7 @@ public final class PackageInfoMapper: PackageInfoMapping {
                     }
 
                     result[target.name] = ModuleMap.custom(moduleMapPath)
+                case .test where testableTargetsFromPackages.contains(packageInfo.key): fallthrough
                 case .regular:
```

Comment:
Why not just `,`?
```suggestion
                case .test where testableTargetsFromPackages.contains(packageInfo.key), .regular:
```

Reply:
Yea could be a comma `,`. No strong preference here

Reply:
fixed